### PR TITLE
Announcement of community call

### PR DESCRIPTION
**Pull request by naoe-tatara**
Ref. https://hackmd.io/Nhgx1ss5Q7mvQ8ua-ZBadQ?both#Action-points
Please improve if anything is incorrect or missing :)

### CODE REVIEW

**Review by blankdots — APPROVED** [review]

File: tweets/YYYY/MM/ama-announce-01.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Interested in helping, teaching, hosting a CodeRefinery workshop, contributing to a lesson material, using GitLab service, etc.? Join us in a community call "Ask Me Anything" on Aug 23, 12:00 CEST! Connection details will be announced later.
```
Comment (bast) [inline tweets/YYYY/MM/ama-announce-01.tweet:1]:
```suggestion
Interested in helping, teaching, hosting a CodeRefinery workshop, contributing to lesson material, using GitLab service, etc.? Join us in a community call "Ask Me Anything" on Aug 23, 12:00 CEST! Connection details will be announced later.
```

**Comment by bast** [comment]
Thanks! Is it "Ask Me Anything" or "Ask Us Anything"? (I am not sure, honest question)

**Comment by naoe-tatara** [comment]
Great point! Yes, it should be ask 'us' anything :)

File: tweets/YYYY/MM/ama-announce-01.tweet (hunk 1 above)
Comment (naoe-tatara) [reply tweets/YYYY/MM/ama-announce-01.tweet:1]: Not sure how to merge from mobile, but thanks for the sui!

**Review by bast — APPROVED** [review]

**Comment by bast** [comment]
Just for the record: the approval was "dismissed" because a change invalidated it. All is good.

**Comment by bast** [comment]
Maybe I will merge tomorrow so that it shows up in the timelines during day time? Not sure what the best timing is.

**Comment by naoe-tatara** [comment]
> Maybe I will merge tomorrow so that it shows up in the timelines during day time? Not sure what the best timing is.

I think it is good idea to merge during the day time :)

**Pull request merged**